ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FunctionalOrMentalStatus-01.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<adaxml>
+    <meta status="new" created-by="annevanderkant" last-update-by="annevanderkant"
+        creation-date="2021-03-01T14:59:12.254+01:00"
+        last-update-date="2021-03-01T14:59:12.254+01:00"/>
+    <data>
+        <functionele_of_mentale_status_registratie app="zib2020-nl" shortName="functionele_of_mentale_status_registratie" formName="functionele_of_mentale_status" transactionRef="2.16.840.1.113883.2.4.3.11.60.121.4.2.4.26" transactionEffectiveDate="2020-09-01T00:00:00" versionDate="" prefix="zib2017-" language="nl-NL" title="FunctioneleOfMentaleStatus-01" desc="" id="74e48aae-0924-4db7-a48a-ae0924adb775">
+            <functionele_of_mentale_status conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.1">
+                <onderwerp conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.1">
+                    <patient-id value="111222333" root="2.16.840.1.113883.2.4.6.3" conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.2"/>
+                </onderwerp>
+                <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">
+                    <medisch_hulpmiddel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.1" id="mh-1">
+                        <product conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.2">
+                            <product_type code="462777007" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED Clinical Terms" displayName="elektrisch scheerapparaat" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.3"/>
+                        </product>
+                        <product_omschrijving value="Ergonomisch elektrisch scheerapparaat" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.13"/>
+                        <begin_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.11"/>
+                    </medisch_hulpmiddel>
+                </referenties>
+                <status_naam code="284888003" codeSystem="2.16.840.1.113883.6.96" displayName="kan zich scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.2"/>
+                <status_waarde code="284892005" codeSystem="2.16.840.1.113883.6.96" displayName="moeite met scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.3"/>
+                <status_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.6"/>
+                <toelichting value="Dit is een toelichting" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.4"/>
```

Comment:
Could we include a more speaking comment, such as 'Door verminderde fijne motoriek niet mogelijk om het gezicht volledig glad te scheren'

Reply:
Done!